### ISSUE TITLE
RasterAnalytics fileshare template fails to registerd RasterStore

### ISSUE THREAD

**Issue by hmelendez** [Esri/arcgis-powershell-dsc#117]
I'm using the RasterAnalytics-fileshare template to create a site, it creates the FileShareLocalPath fileshare but it fails to register it as a RasterStore with the site an then it can not set the new site as a RasterAnalytics on the portal. This are the parameters that i'm using and bellow is the error return by DSC.

```
"DataStoreItems":  {    
                                 "RasterStore":  {
                                           "FileShareName":  "rasterdatastore",
					   "FileShareLocalPath":  "F:\\arcgisserver\\directories\\rasterdatastore" }}
```

> Cannot bind argument to parameter 'FilePath' because it is an empty string.
The PowerShell DSC resource '[ArcGIS_FileShare]RasterAnalysisFileShare' with SourceInfo 'C:\Program Files\WindowsPowerShell\Modules\ArcGIS\1.1.4\Configuration\ArcGISConfigure.ps1::1168::21::ArcGIS_FileShare' threw one or more non-terminating errors while running the Set-TargetResource functionality. These errors are logged to the ETW channel called Microsoft-Windows-DSC/Operational. Refer to this channel for more details.
The SendConfigurationApply function did not succeed.

**Comment by shailesh91** [comment]
@hmelendez this issue is being caused by an empty string binding exception caused by the line below
https://github.com/Esri/arcgis-powershell-dsc/blob/0fa1ca8dcadaaa83b791bf48468787458a76dd3c/Modules/ArcGIS/DSCResources/ArcGIS_FileShare/ArcGIS_FileShare.psm1#L102
This will be fixed in the next version that will be available in last week of Feb or First week on March.

**Comment by mickhuska** [comment]
Any updates on this bug? Currently experiencing the same issue.

**Comment by shailesh91** [comment]
@mickhuska A fix is coming next week with release v2 along with support for 10.7.

**Comment by shailesh91** [comment]
Closing this issue. Fixed with release v2.0.0!
